### PR TITLE
Integrate Dashboard journey for Composable Modelbank POC

### PR DESCRIPTION
**Pull request by stasprigodich**

### CODE REVIEW

**Review by burgbart — CHANGES_REQUESTED** [review]

File: Podfile [hunk 1]
```diff
@@ -35,6 +35,14 @@ abstract_target 'Common' do
   pod 'BackbaseDesignSystem', '4.2.2'
   pod 'UserManagerUserProfileUseCase'
   pod 'BackbaseObservability', '~> 1.0'
+  pod 'RetailAccountsAndTransactionsJourney', '~> 5.4.0'
+  pod 'RetailAccountsAndTransactionsJourneyAccountsUseCase', '~> 5.4.0'
+  pod 'RetailAccountsAndTransactionsFinancialInstitutionsUseCase', '~> 5.4.0'
+  pod 'RetailAccountsAndTransactionsArrangementViewsUseCase', '~> 5.4.0'
+  pod 'RetailAccountsAndTransactionsJourneyLoansUseCase', '~> 5.4.0'
+  pod 'RetailAccountsAndTransactionsJourneyTransactionsUseCase', '~> 5.4.0'
+  pod 'RetailAccountsAndTransactionsJourneyExternalAccountAggregationUseCase', '~> 5.4.0'
```
Comment (7yon) [inline Podfile:44]: Shall we remove unused use cases for now?
- RetailAccountsAndTransactionsFinancialInstitutionsUseCase
- RetailAccountsAndTransactionsJourneyLoansUseCase
- RetailAccountsAndTransactionsJourneyExternalAccountAggregationUseCase

File: app/Extensions/AppDelegate+DashboardJourney.swift [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+import Resolver
+import DashboardJourney
+import RetailAccountsAndTransactionsJourney
+import RetailAccountsAndTransactionsJourneyAccountsUseCase
+import RetailAccountsAndTransactionsArrangementViewsUseCase
+import RetailAccountsAndTransactionsJourneyTransactionsUseCase
+
+// MARK: - Dashboard Journey methods
+extension AppDelegate {
```
[the remark this answers is not in the document]
Comment (bdomok) [reply app/Extensions/AppDelegate+DashboardJourney.swift:13]: I'm not sure that's justified for iOS, in my experience it's usually 1 file per journey. In my [PR](https://github.com/Backbase/golden-sample-app-ios/pull/14/files#diff-345e5d826e2411eb14b7c6c36ce00bcef9c0835a6cc2442eebaf7203a18858a9) I moved these "config files" out from extensions to "Dependencies". What do you think about that?

File: app/Factories/DashboardTileFactory.swift [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        }
+    }
+
+    private func checkEntitlements(_ entitlements: [DashboardCore.Entitlement]) -> Bool {
+        guard !entitlements.isEmpty else { return true }
+        return !Self.allowedFeatures.filter { entitlements.contains($0) }.isEmpty
+    }
+
+    // MARK: - Methods
+    func makeLatestTransactionsTile(_ parentNavigationController: UINavigationController) -> (any TileProtocol)? {
```
Comment (bdomok) [inline app/Factories/DashboardTileFactory.swift:54]: These factories can be static as well to remove the need of creating an instance of this factory.

File: Podfile (hunk 1 above)
Comment (stasprigodich) [reply Podfile]: @7yon fixed

File: app/Extensions/AppDelegate+DashboardJourney.swift (hunk 2 above)
[the remark this answers is not in the document]
Comment (stasprigodich) [reply app/Extensions/AppDelegate+DashboardJourney.swift:13]: @burgbart could you take a look, can it be done in a separate Balazs's PR?

**Review by 7yon — APPROVED** [review]

**Pull request closed without merging**